changelog/WGyqIUL3Se6bP0_yWYR6Jw.md

```diff
@@ -0,0 +1,2 @@
+level: silent
```

Comment:
why silent if the difference is in the UI?

Reply:
From https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md#levels
> * silent: Changes to tests or documentation
> * silent: Minor changes or fixes to UI

Reply:
I wonder if I'll ever get used to that scale.... somehow everything is 2 tones down in there comparing to how I'd personally classify changes 😄